extensions/azurecore/src/account-provider/azureAccountProvider2.ts

```diff
@@ -305,7 +305,9 @@ export class AzureAccountProvider implements azdata.AccountProvider {
 
 		tenants = tenants.filter(t => t !== undefined);
 		if (tenants.length === 0) {
-			throw new Error(localize('azure.noTenants', "No azure tenants found. Failing..."));
+			const msg = localize('azure.noTenants', "No azure tenants found. Failing...");
```

Comment:
I'd also suggest looking through the other errors and think about displaying some of them to the user as well. For example https://github.com/microsoft/azuredatastudio/blob/master/extensions/azurecore/src/account-provider/azureAccountProvider2.ts#L287 is probably something we want to show